ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -233,14 +226,30 @@ public class BottomSheetController: UIViewController {
         }
     }
 
+    private func updateExpandedContentAlpha() {
+        let transitionLength = Constants.expandedContentAlphaTransitionLength
+        let currentOffset = currentOffsetFromBottom
+        let collapsedOffset = collapsedOffsetFromBottom
+
+        var targetAlpha: CGFloat = 1.0
+        if currentOffset <= collapsedOffset {
+            targetAlpha = 0.0
+        } else if currentOffset > collapsedOffset && currentOffset < collapsedOffset + transitionLength {
+            targetAlpha = abs(currentOffset - collapsedOffsetFromBottom) / transitionLength
+        }
+        expandedContentView.alpha = targetAlpha
+    }
+
     @objc private func handlePan(_ sender: UIPanGestureRecognizer) {
         switch sender.state {
         case .began:
             stopAnimationIfNeeded()
+            hostedScrollView?.setContentOffset(.zero, animated: true)
```

Comment:
doesn't it make more sense to set the content offest to zero at the end of the animation? or am I missing osomethign?

Reply:
Yup that does make more sense, I changed it to only reset the scrollview after the sheet collapses.